gulpfile.js

```diff
@@ -1,201 +1,196 @@
 var gulp = require('gulp'),
-sass = require('gulp-sass'),
-autoprefixer = require('gulp-autoprefixer'),
-cssnano = require('gulp-cssnano'),
-jshint = require('gulp-jshint'),
```

Comment:
Removed reference to unused plugins (jshint, imagemin, notify, cache, livereload)